### DISCUSSION TITLE
Nfo option?

### DISCUSSION THREAD

**Discussion by jimlynnjulian** [jellyfin/jellyfin#7848] [General] [answered]
During the Jellyfin library setup, there is a checkbox for the metadata filetype used in saving metadata.
What happens if the box is not checked? What filetype is used?
I just tried with a test library. If the data is not in an Nfo file, where is the data kept? I checked the Appdata folders and the Jellyfin/Server folders and could not determine the location.
Thanks.

**Comment by crobibero** [comment, marked as answer]
The data is always kept in the database. If you enable NFO files the data will be replicated to them.

**Reply by jimlynnjulian** [reply]
I understand. Thyanks. However, when I try to edit the library's settings, the form that shows up is not the same as the form created when the library was first entered. 

I located the options.xml file at:
"C:\ProgramData\Jellyfin\Server\root\default\N\N.xml"
Thanks.
